src/aleph/netty.clj

```diff
@@ -1384,11 +1384,17 @@ initialize an DnsAddressResolverGroup instance.
    (.newHandler ssl-ctx
                 (.alloc ch)))
   (^ChannelHandler
-   [^Channel ch ^SslContext ssl-ctx ^InetSocketAddress remote-address]
-   (.newHandler ssl-ctx
-                (.alloc ch)
-                (.getHostName ^InetSocketAddress remote-address)
-                (.getPort ^InetSocketAddress remote-address))))
+   [^Channel ch ^SslContext ssl-ctx ^InetSocketAddress remote-address ssl-endpoint-id-alg]
```

Comment:
I would create another arity instead.
In the current state, it might be a breaking change for some people.

Reply:
Good catch, I didn't realize that it's part of the public API. It also made me notice that the docstring doesn't match the implementation anymore :grimacing: 

I've pushed a commit to reinstate the 3-ary version and update the docstring. I also pushed another one to change the return type to `SslHandler` - seemed like a reasonable refinement.